pillar/edx/ansible_vars/xpro.sls

```diff
@@ -13,19 +13,19 @@
 {% set heroku_env = heroku_xpro_env_url_mapping['{}'.format(purpose)] %}
 
 edx:
-  JWT_ISSUER: 'OAUTH_OIDC_ISSUER'
-  JWT_AUDIENCE: '{{ business_unit }}-{{ environment }}-key'
-  JWT_SECRET_KEY: __vault__:gen_if_missing:secret-{{ business_unit }}/{{ environment }}/jwt-secret-key>data>value
-  JWT_SIGNING_ALGORITHM: 'RS512'
-  JWT_PRIVATE_SIGNING_JWK: __vault__::secret-{{ business_unit }}/{{ environment }}/jwt-signing-jwk/private-key>data>value
-  JWT_PUBLIC_SIGNING_JWK_SET: __vault__::secret-{{ business_unit }}/{{ environment }}/jwt-signing-jwk/public-key>data>value
   ansible_vars:
     EDXAPP_EDX_API_KEY: __vault__:gen_if_missing:secret-{{ business_unit }}/{{ environment }}/edx-api-key>data>value
     EDXAPP_SESSION_COOKIE_DOMAIN: .xpro.mit.edu
     EDXAPP_SESSION_COOKIE_NAME: {{ environment }}-{{ purpose }}-session
     EDXAPP_COMMENTS_SERVICE_URL: "http://localhost:4567"
     EDXAPP_COMMENTS_SERVICE_KEY: __vault__:gen_if_missing:secret-{{ business_unit }}/global/forum-api-key>data>value
     EDXAPP_IDA_LOGOUT_URI_LIST: ['{{ heroku_env }}/logout']
+    EDXAPP_LMS_ISSUER: {{ heroku_env }}/oauth2
```

Comment:
Should this be the xPro app, or the LMS?

Reply:
Good catch. Fixed